modules/RTC/TraceablePeerConnection.js

```diff
@@ -143,7 +143,7 @@ export default function TraceablePeerConnection(
      * The map holds remote tracks associated with this peer connection.
      * It maps user's JID to media type and remote track
      * (one track per media type per user's JID).
-     * @type {Map<string, Map<MediaType, JitsiRemoteTrack>>}
+     * @type {Map<string, Map<MediaType, Set<JitsiRemoteTrack>>>}
```

Comment:
the changes in this file are more production ready by adding support for storing multiple remote tracks per media type